### PR TITLE
[Merged by Bors] - document resource limits

### PR DESCRIPTION
**Pull request by razvan**

### CODE REVIEW

File: docs/modules/ROOT/pages/usage.adoc [hunk 1]
```diff
@@ -130,52 +130,14 @@ For a full list of configuration options we refer to the HBase https://hbase.apa
...
-        resources:
-          cpu:
-            max: '500m'
-            min: '250m'
-----
-
-=== Defaults
-
-If nothing is specified, the operator will automatically set the following default values for resources:
+If no resources are configured explicitely, the Hbase operator uses following defaults:
```
Comment (maltesander) [inline docs/modules/ROOT/pages/usage.adoc:140]: I think i forgot to correct that in airflow if that is copy paste.
```suggestion
If no resources are configured explicitly, the HBase operator uses following defaults:
```

**Review by maltesander — APPROVED** [review]
LGTM! Should we add changing to stable to the release procedure docs?

**Comment by razvan** [comment]
yeah, sure.

**Comment by razvan** [comment]
bors merge

**Pull request closed without merging**